go/extractor/util/util.go

```diff
@@ -281,3 +282,17 @@ func EscapeTrapSpecialChars(s string) string {
 	s = strings.ReplaceAll(s, "#", "&num;")
 	return s
 }
+
+func FindGoFiles(root string) bool {
+	found := false
+	filepath.WalkDir(root, func(s string, d fs.DirEntry, e error) error {
+		if e != nil {
+			return e
+		}
+		if filepath.Ext(d.Name()) == ".go" {
+			found = true
+		}
```

Comment:
Does this continue walking through files in the directory even if the first thing it finds is a `.go` file? If so, is there any way to get it to stop once you have found something?

Reply:
Indeed, it now returns `filepath.SkipAll`, which does exactly that.